affiliated/index.html

```diff
@@ -255,6 +255,15 @@ <h1 id="becoming-an-affiliated-package">Becoming an Affiliated Package<a class="
 		consistent with basic principles of open source. (E.g.,
 		<a href="https://choosealicense.com/">an OSI-approved license</a>)</li>
 
+		<li>Include instructions to users on how to cite your package. This is commonly done
+			with a CITATION file. This file could include acknowledgement text, a Zenodo record
+			(highly recommended), and/or journal article(s). Where possible, full BibTeX entries of these citations
```

Comment:
Nitpick: Highly recommended option should be listed first.
```suggestion
			with a CITATION file. This file could include a Zenodo record (highly recommended),
			acknowledgement text, and/or journal article(s). Where possible, full BibTeX entries of these citations
```